src/FuseProcess.ts

```diff
@@ -22,6 +23,40 @@ export class FuseProcess {
         return this;
     }
 
+		require(close?: ((exports: any) => void)|(() => void)) {
+			function getMainExport(mdl) {
+				return mdl && mdl.FuseBox && mdl.FuseBox.mainFile ?
+						mdl.FuseBox.import(mdl.FuseBox.mainFile) :
```

Comment:
I don't think we need to import it manually. Once a bundle is loaded it's gets executed right away, unless you want to get the results?!